WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -105,6 +105,11 @@
     public static final String WORDPRESS_PURCHASES_URL = "https://wordpress.com/purchases";
 
     /**
+     * url for redirecting free users to empty their sites (start over)
+     */
+    private final String WORDRESS_EMPTY_SITE_SUPPORT_URL = "https://en.support.wordpress.com/empty-site/";
```

Comment:
Just a note: I find it odd that this field is used in the exact same manner as the ones above, yet it produces a warning because it's private.